tests/test_delete_query.py

```diff
@@ -0,0 +1,31 @@
+import json
+
+import pytest
+
+from snuba_sdk.delete_query import DeleteQuery
+
+queries = [
+    pytest.param(
+        DeleteQuery(
+            storage_name="search_issues",
+            column_conditions={"project_id": [1], "occurrence_id": ["1234"]},
+        ),
+        '{"columns":{"project_id": [1], "occurrence_id": ["1234"]}}',
+    ),
+    pytest.param(
+        DeleteQuery(
+            storage_name="non-real-storage",
+            column_conditions={},
+        ),
+        '{"columns":{}}',
```

Comment:
heres a practical example of my decision not to do validation at the sdk level.

the client doesnt include `project_id`, since theres no validation in the sdk it fine here, and the client will get 400 from the endpoint for missing project_id. If the requirements for the endpoint ever change its just 1 change at the endpoint.

The alternative where the sdk does validation would require project_id here, but if that ever changes it will require a code change at the endpoint as well as here. Which is why i think its better for validation to only happen at the endpoint.